### DISCUSSION TITLE
"●" masking issue when setting FontFamily of PasswordBox control in Non-Windows.

### DISCUSSION THREAD

**Discussion by luckypong** [unoplatform/uno#14822] [Q&A]
hello.

I'm currently developing with WinUI3 and testing with a non-Windows project, and I'm asking about a masking issue with PasswordBox.

When FontFamily is set to a non-English font and the visual -state becomes normal after entering the password, masking disappears.

The test was conducted in two styles.
1) Default Style
     - Control Style: DefaultPasswordBoxStyle
     - Control FontFamily: non-English(Unicode) OTF Font.
2) Custom Style
     - Control Style: DefaultPasswordBoxStyle2
     -Setter FontFamily: XamlAutoFontFamily
     - PlaceholderTextContentPresenter FontFamily: non-English(Unicode) OTF Font

Is this a bug? Or 2) Please answer whether I should create a Style and use it.
Or is there a problem on MS Xaml side?

https://github.com/unoplatform/uno/assets/11326846/20560952-5398-4bbf-ac97-38392c304043

**Comment by jeromelaban** [comment]
Thanks for the report!

Could you tell which platform you're testing with?

**Reply by luckypong** [reply]
wasm
 - PasswordBox
      Normal - not showing masking.
      Focused - not showing masking.
 - ContentDialog
      Title - not showing string.
 - NavigationView
      Header - showing default fontfamily.

wpf, gtk
  - PasswordBox
      Normal - not showing masking.
      Focused - showing masking.
  - ContentDialog, NavigationView
      Title,Header - showing squares not string.

android
  - Android seems to be fine.

ios, mac
  - I couldn't test iOS because I don't have a device, but it seems to be the same as Android.

**Reply by jeromelaban** [reply]
Thanks. This may look incorrect. Can you provide [the repro sample](https://platform.uno/docs/articles/uno-howto-create-a-repro.html) you've been using to test this and [open an issue](https://github.com/unoplatform/uno/issues) with it?

**Reply by luckypong** [reply]
The font family is Korean.
Attached sample AppUno5.
thank you.

[UnoApp5.zip](https://github.com/unoplatform/uno/files/13685579/UnoApp5.zip)

**Reply by jeromelaban** [reply]
Thanks, make sure to open an issue with details on how to test the sample app.
